app/views/pages/postcode.html.erb

```diff
@@ -0,0 +1,40 @@
+<body>
+<ul>
+  <% @stops.each do |stop| %>
+    <li>
+      <h2><%= stop["name"] %> </h2>
+      <table style="border-collapse: collapse; width: 100%;">
```

Comment:
Maybe not too important for an exercise but it's better to keep styles in a separate stylesheet - if it's not too difficult to do it might be worth moving it